cpp/lib/include/Zotero.h

```diff
@@ -214,6 +214,7 @@ struct SiteParams {
     std::vector<std::string> additional_fields_;
     std::vector<std::string> non_standard_metadata_fields_;
     std::unordered_map<std::string, std::unique_ptr<RegexMatcher>> field_exclusion_filters_;
+    std::unordered_map<std::string, std::unique_ptr<RegexMatcher>> field_removal_filters_;
```

Comment:
Do we really anticipate that many map entries that we need an unordered_map?  The same question should be asked of the field_exclusion_filters_ field.

Reply:
Looking at the usage of this field: Do we really need any kind of map or would a vector of pairs suffice?